WordPress/Classes/ViewRelated/Post/Views/PostAuthorPicker.swift

```diff
@@ -0,0 +1,97 @@
+import SwiftUI
+import WordPressData
+import WordPressUI
+import WordPressShared
+
+struct PostAuthorPicker: View {
+    @StateObject private var viewModel: PostAuthorPickerViewModel
+    @State private var searchText = ""
+    @Environment(\.dismiss) private var dismiss
+
+    init(post: AbstractPost, onSelection: @escaping () -> Void) {
+        _viewModel = StateObject(wrappedValue: PostAuthorPickerViewModel(post: post, onSelection: onSelection))
```

Comment:
I presume this is generated by AI? I feel bad about annoying you with the same comments. Any chance we can tell AI not to generate code like this?

Reply:
There are multiple precedents in the codebase where `StateObject(wrappedValue:)` is used. I'd continue using it where it's OK – if it's part of the new screen it's fine.